BackupPC_report

```diff
@@ -8,7 +8,7 @@
 
 use strict;
 use warnings;              # global flag -w produces warnings from BackupPC::Lib
-use lib '/usr/local/lib';  # <<< change path prefix to match your installation
+use lib '/usr/share/BackupPC/lib';  # <<< change path prefix to match your installation
```

Comment:
I understand why you did this, but I am not agree with this change just because it is not convenient for me since I am a FreeBSD user.
Probably the proper way to resolve this issue is to include the `BackupPC_report` script into `BackupPC` code.